source/_posts/2021-07-07-release-20217.markdown

```diff
@@ -80,10 +79,9 @@ Genie you can now change the room size for your diffuser.
 ## Trigger conditions and trigger IDs
 
 If you are creating some complex automations in YAML, you might be familiar with
-this. Consider an big automation, with a whole bunch of triggers. But how
-would you know which of those triggers actually triggered the automation?
+this. Consider a big automation, with a whole bunch of triggers. But how would you know which of those triggers actually triggered the automation?
 
-You can now assign an `id` to your triggers that is passed into automation when
+You can now assign an `id` to your triggers that are passed into automation when
```

Comment:
It is referring to `id` which is singular, thus "is" and not "are".

Reply:
right, sorry.